presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/SuiteList.java

```diff
@@ -37,11 +38,11 @@
 public final class SuiteList
         implements Runnable
 {
-    private final Module additionalSuites;
+    private final Module additionalEnvironments;
```

Comment:
why so?
I guess the modules can add both new suites and new environments. Right?

Reply:
I've corrected this in a subsequent commit. I'll squash those.